modules/planning/tasks/deciders/path_reuse_decider/path_reuse_decider.cc

```diff
@@ -58,17 +60,23 @@ Status PathReuseDecider::Process(Frame* const frame,
                                                 ->mutable_path_reuse_decider();
   ADEBUG << "lane_change_status->is_current_opt_succeed(): "
          << lane_change_status->is_current_opt_succeed();
-  // reuse path when: in change lane status; reuse_path is enable and
-  // optimization is successful
-  if (!lane_change_status->is_current_opt_succeed() ||
-      lane_change_status->status() != ChangeLaneStatus::IN_CHANGE_LANE ||
-      !Decider::config_.path_reuse_decider_config().reuse_path()) {
-    /* increase total path number when fail to reuse due to optimization*/
-    if (lane_change_status->status() == ChangeLaneStatus::IN_CHANGE_LANE &&
-        Decider::config_.path_reuse_decider_config().reuse_path()) {
-      ++total_path_counter_;
-    }
+
+  if (!Decider::config_.path_reuse_decider_config().reuse_path() ||
+      lane_change_status->status() != ChangeLaneStatus::IN_CHANGE_LANE) {
     ADEBUG << "skipping reusing path";
+    return Status::OK();
+  }
+
+  /*count total_path_ when in_change_lane && reuse_path*/
+  ++total_path_counter_;
+
+  /*reuse path when in non_change_lane reference line or
+    optimization succeeded in change_lane reference line
+  */
+  if (!is_change_lane_path ||
+      (is_change_lane_path && lane_change_status->is_current_opt_succeed())) {
```

Comment:
`is_change_lane_path && ` is not needed.

Reply:
Fixed. Thank you!